layouts/page/superpowers.html

```diff
@@ -0,0 +1,276 @@
+{{ define "hero" }}
+    <header class="header-hero shadow-2xl relative overflow-hidden">
```

Comment:
I feel like this page would benefit from one of these nav banners that we use on other pages, to jump to sections:

![image](https://user-images.githubusercontent.com/3953235/79487164-a9f9d580-7fcc-11ea-9f2d-89eccc7e8d4d.png)

If we did this, I would imagine the navigation links being something like SUPERPOWERS, LAUNCH EVENT, WATCH THE TALKS.

Reply:
Agreed - advancing to each section will make stuff below the fold more discoverable.